app/pandas.ipynb

```diff
@@ -208,18 +429,93 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[\"Average age\"] = df[\"Age\"].mean()",
```

Comment:
You have added the 'Average age' column twice. This results in redundancy in your DataFrame. Please remove the duplicate column to streamline your data.